src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -125,7 +127,7 @@ public void AnalyzeDirectory_DirectoryWithInvalidTemplates_LogsExpectedErrorInSa
             Assert.AreEqual(templateErrorMessage, toolNotifications[1]["message"]["text"]);
 
             var nonJsonFilePath1 = Path.Combine(directoryToAnalyze, "AnInvalidTemplate.json");
-            var nonJsonFilePath2 = Path.Combine(directoryToAnalyze, "AnotherInvalidTemplate.json");
```

Comment:
Why did you remove this file? It was to test the code that shows multiple paths with errors

Reply:
My intention is that this file was replaced with AnInvalidTemplate.bicep , does this not provide the same coverage?

Reply:
Oh, you're right, never mind